docs/char-info-str.md

```diff
@@ -30,7 +30,7 @@ api_type:
 - HeaderDef
 ---
 
-# CHAR\_INFO structure
+# CHAR_INFO structure
```

Comment:
Don't these major headers also need to be like `CHAR_INFO` structure or risk getting mistranslated?

Reply:
Right, I was concerned about this too. Since the translation will apply to all un-backquoted text, what guarantees that **this** will not be translated into.. well, whatetver the translations are for "Char" and "Info" in any given language? This is a reference to `CHAR_INFO` the structure in our API

Reply:
We can actually check just check this now since the pages have been translated. :) 
I _think_ it looks okay if I'm understanding the issue you're pointing out correctly -- here is the page in French: https://docs.microsoft.com/fr-fr/windows/console/char-info-str

Reply:
_Sorta_??

![image](https://user-images.githubusercontent.com/189190/102670326-20df0380-4144-11eb-8578-ace71c064f2e.png)
![image](https://user-images.githubusercontent.com/189190/102670412-26d4e480-4144-11eb-9055-e128646ba7ab.png)

Absolutely not

![image](https://user-images.githubusercontent.com/189190/102670562-305e4c80-4144-11eb-9711-7164fd6d6f33.png)

Reply:
I see what you mean. We need to have any actual code like this in backticks so that they know not to translate it.

Reply:
I can fix this instance and search for any other examples of Char_ that aren't already in a code fence. 
Any ideas about how to find other examples where there may be proper code names like this getting translated?

Reply:
This is a good question. For stuff like this, our structures usually don't have lowercase names in them. You could safely do a regular expression search for `[A-Z0-9_]{5,}` to find instances of them (5 chars or more, uppercase with underscores/numbers) and manually filter out the ones that aren't enclosed in backticks. Warning though, there's a ton.

I am more worried about the YAML front matter. It doesn't follow Markdown syntax, right? How do we handle `title: CHAR_INFO struture` on line 2 of this document? They shouldn't translate the page title either.

Reply:
Looks like many other examples are in `*-str.md`, both in the front matter and the first heading.

Reply:
Some things have _double backticks_ (line 25, console-structures.md in this PR)